docs/contributing.md

```diff
@@ -185,20 +188,20 @@ the browser will automatically show the new content.
 
 Create a `puml` file inside `docs/diagrams/src`
 
-```
+```sh
 touch docs/diagrams/src/demo.puml
 ```
 
 Check the PlantUML syntax here: <https://plantuml.com/deployment-diagram>
 
 Serve docs site on localhost:
 
-```
+```sh
 mkdocs serve
 ```
 
 Add the yielded `svg` into the desired `.md` file
 
-```
-![Diagram demo](diagrams/out/demo.svg "Diagram demo")
-```
+```markdown
+![Diagram demo](diagrams/out/edpm.svg "Diagram EDPM")
+```
```

Comment:
The idea here was more to show the syntax, but it is confusing, maybe it is better to remove this